### PR TITLE
Update CONTRIBUTORS

### PR DESCRIPTION
**Pull request by jaboc83**

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -7,3 +7,4 @@ Jordan Adler
 Kent Brewster
 Malte Ubl
 Taylor Savage
+Jake Moening
```
Comment (erwinmombay) [inline CONTRIBUTORS]: mind sorting :smile_cat:

**Comment by erwinmombay** [comment]
@jaboc83 sorry, just 1 more request. can you squash into 1 commit

**Comment by jaboc83** [comment]
yep sorry, I was using the git editor for these. Just got home. I'll squash it :)

**Comment by erwinmombay** [comment]
@jaboc83 np. thanks!

**Pull request merged**